src/localization/en.json

```diff
@@ -258,7 +260,7 @@
     "edit": "Edit",
     "delete": "Delete",
     "status": "Status",
-    "confirmDeleteTitle": "Are you sure to delete?",
+    "confirmDeleteTitle": "Are you sure you want to delete?",
```

Comment:
How about using the format string option to insert the name of the questionnaire to be deleted into this string, rather than having it separate in the Dialog content.
Message could be "Are you sure you want to delete questionnaire: [questionnarieTitle]?"

Or even shorter:
"Delete [questionnaireTitle]?"

You can see an example of how this is done in the timebank.atTheEndOf string. You will also need to look where in the code this is used to see the syntax for passing the title.

Reply:
Would also need to be updated in the fi.json